content/posts/trans_pride/index.md

```diff
@@ -0,0 +1,88 @@
+
+I know things feel heavy sometimes, and right now you might feel soo many things at once -- overwhelmed, uncertain, or even scared about
+what's happening to the world right now. I feel that too. As a baby-transfem, I'm walking this path beside you.
+It's okay to be scared. Who wouldn't be in times like these? It's okay to feel like you don't have all the answers or
+don't know what to do next - like everything feels like it's too much at once. We're all figuring this out as we go,
+and that's absolutely fine.
+
+There is something I want you to remember. Write it down if you have to: **You are valid. You are enough.
+You are wonderful just the way you are.**<br>
+Whether you're trans, nonbinary, genderfluid, cis or questioning -- whether you're gay, bi, pan, ace, or anywhere in
+between or outside this spectrum -- we are all part of this wonderful and diverse community. Every single one of us
+deserves love, respect and space to exist fully and freely.
+
+During Trans Awareness Month, we're given opportunity to reflect, honor and amplify the voices of trans and gender-expansive
+folks. But it's not just about awareness -- it's about **action**. It's about supporting one another, standing up for one another,
+and doing whatever we can to make the world a safer, more inclusive space for all of us. Whether by speaking out about injustice,
+donating to trans*-organizations, or simply checking in on each other. In times like these we all need a little extra love.
+Our collective action matters. Big and small, loud and quiet. We have to stick together now more than ever. Because when one is hurt,
+we can all feel it. And when one of us thrives, we all do.
+
+I know things might feel bleak right now. The world and society can feel like it's turning on us, and the weight of it can be a lot to carry.
+But, friend, I want you to remember something else: **It is okay to not be okay**. You don't have to have everything figured out right now.
+You don't have to carry the burden of the world on your shoulders. It's okay to have moments of doubt, of frustration, of pain.
+But please remember: **You are loved so much**. You are seen. You are precious. *And yes, things will get better*.
+
+As someone, who has only recently discovered her true self, I often have to remind myself of that too.
+Sometimes, I sit in these holes of agony not understanding what is going on, fearing the future. But it's important to reach out for help. I'm never alone. **You are never alone**.
+We are all here for each other, whether we've known each other for years or we've just met. There's a strength in our community.
+And that strength doesn't disappear just because some orange and old people won an election. They are out of touch with reality.
+
+So, as we celebrate Trans Awareness Month together, take a moment to breathe. Take a sip of your hot chocolate or tea -- or whatever it is you are drinking -- and allow yourself
```

Comment:
Maybe shorten it a bit?

```suggestion
So, as we celebrate Trans Awareness Month together, take a moment to breathe. Take a sip of your preferred beverage and allow yourself
```

Reply:
Yep I understand where you're coming from. Idk though. The post is supposed to feel like a friend is talking to the reader or sends a voice message and I think this style fits that goal quite well.
But I'll look if I can maybe shorten that while keeping the original feeling. Thanks for the review ^^